biomage/stage/stage.py

```diff
@@ -21,6 +22,7 @@
 
 DEFAULT_BRANCH = "master"
 
+yaml = YAML()
```

Comment:
`yaml` is only used inside the `get_manifests` function. Can you move the instance creation to happen there inside that function?